kubetest2-gke/deployer/commandutils.go

```diff
@@ -150,3 +154,36 @@ func resolveLatestVersionInChannel(loc, channelName string) (string, error) {
 
 	return "", fmt.Errorf("channel %q does not exist in the server config", channelName)
 }
+
+func containerArgs(args ...string) []string {
+	return append(append([]string{}, "container"), args...)
+}
+
+func runWithNoOutput(cmd exec.Cmd) error {
+	exec.NoOutput(cmd)
+	return cmd.Run()
+}
+
+func runWithOutput(cmd exec.Cmd) error {
+	exec.InheritOutput(cmd)
+	return cmd.Run()
+}
+
+func runWithOutputAndBuffer(cmd exec.Cmd) (string, error) {
```

Comment:
On my second thought this function is probably better to be named as `runWithOutputAndReturn`. It's a bit verbose but I'm not able to come up with a better name...

Reply:
Done